app/webrtc/client/actionLink.js

```diff
@@ -0,0 +1,29 @@
+import { Session } from 'meteor/session';
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
+import toastr from 'toastr';
+
+import { actionLinks } from '../../action-links/client';
+import { APIClient } from '../../utils/client';
+import { Messages } from '../../models';
+
+actionLinks.register('joinLivechatWebRTCCall', function(message) {
+	if (Session.get('openedRoom')) {
+		const updatedMessage = Messages.findOne({ _id: message._id });
+		if (updatedMessage.callStatus === 'declined' || updatedMessage.callStatus === 'ended') {
+			toastr.info(TAPi18n.__('Call Already Ended', ''));
+		} else {
```

Comment:
Same goes here. Try to avoid unnecessary else statements
https://github.com/RocketChat/Rocket.Chat/pull/22689/files#r672102416